.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: go
 
+dist: xenial
+
 go:
   - 1.10.x
 
+env:
+  global:
+    - RUNC_VERSION=v1.0.0-rc5
+    - CRIO_VERSION=v1.11.5
+
 sudo: required
 
 before_install:
+  # checkers
   - go get -v github.com/golang/lint/golint
   - go get -v github.com/fzipp/gocyclo
   # hack for building on forks
   - repo=`basename $PWD`; src=`dirname $PWD`; dest="`dirname $src`/intel"
   - if [[ "$src" != "$dest" ]]; then mv "$src" "$dest"; cd ../../intel/$repo; export TRAVIS_BUILD_DIR=`dirname $TRAVIS_BUILD_DIR`/$repo; fi
+  # install buildah build deps
+  - cdir=$(pwd)
+  - sudo apt-get -y install software-properties-common
+  - sudo add-apt-repository -y ppa:alexlarsson/flatpak
```

Comment:
what is available in this PPA that is not available from main distro ?

Reply:
libostree as far as I remember